src/map/enmity_container.cpp

```diff
@@ -139,16 +139,16 @@ void CEnmityContainer::UpdateEnmity(CBattleEntity* PEntity, int16 CE, int16 VE,
         if (enmity_obj->second.PEnmityOwner == nullptr) enmity_obj->second.PEnmityOwner = PEntity;
         float bonus = CalculateEnmityBonus(PEntity);
 
-        int16 newCE = (int16)(enmity_obj->second.CE + (CE > 0 ? CE * bonus : CE));
-        int16 newVE = (int16)(enmity_obj->second.VE + (VE > 0 ? VE * bonus : VE));
+        int32 newCE = (int32)(enmity_obj->second.CE + CE > 0 ? CE * bonus : CE);
```

Comment:
I think this removing these parentheses busted order of operations. here

Reply:
This is a good catch. Fixing it now.